modified:   index.html

slav-mirol opened the pull request:
modified:   index.less
	new file:   package-lock.json
	new file:   src/base/app.less
	new file:   src/base/base.less
	new file:   src/base/container.less
	new file:   src/base/variables.less
	new file:   src/footer.css
	new file:   src/grids/flex-grid.less
	new file:   src/grids/grid.css
	new file:   src/grids/grid.less
	new file:   src/header copy.less
    new file:   src/header.less
	new file:   src/main copy.less
	new file:   src/main.css
	new file:   src/main.less
	new file:   src/utils/animation.less
	new file:   src/utils/box-shadow.less
	new file:   src/utils/button.less
	new file:   src/utils/media.less
	new file:   src/utils/text.less

slav-mirol committed:
modified:   index.html
	modified:   index.less
	new file:   package-lock.json
	new file:   src/base/app.less
	new file:   src/base/base.less
	new file:   src/base/container.less
	new file:   src/base/variables.less
	new file:   src/footer.css
	new file:   src/grids/flex-grid.less
	new file:   src/grids/grid.css
	new file:   src/grids/grid.less
	new file:   src/header copy.less
    new file:   src/header.less
	new file:   src/main copy.less
	new file:   src/main.css
	new file:   src/main.less
	new file:   src/utils/animation.less
	new file:   src/utils/box-shadow.less
	new file:   src/utils/button.less
	new file:   src/utils/media.less
	new file:   src/utils/text.less

m-somov commented on index.html:
```diff
@@ -9,16 +9,321 @@
+              <div class="spisok">
+                <div class="elemOfList">
+                  <div class="rectangle"></div>
+                  <p style="margin-top: 0">
```
Почему бы не заменить инлайн стиль классом?

m-somov commented on index.html:
```diff
@@ -9,16 +9,321 @@
+                    Вы студент или выпускник технических вузов
+                  </p>
+                </div>
+                <div class="elemOfList">
```
Почему тут camelCase? Отличается от стиля остального проекта

slav-mirol replied:
Поменял название класса на elem_of_list

m-somov commented on index.html:
```diff
@@ -9,16 +9,321 @@
+              </div>
+            </section>
+            <section class="col col_md-4 col_sm-12 col_xs-12">
+              <div class="spisok">
```
Все же транслитом писать не стоит

slav-mirol replied:
Поменял название класса на list

m-somov commented on index.html:
```diff
@@ -9,16 +9,321 @@
+            </div>
+
+            <div class="wrapper row">
+              <section class="col col_md-6 col_sm-12 col_xs-12">
```
По условию задачи для планшетов:  если блок на макете имеет 2 колонки то 2 колонки и остаются.  Тут должно быть col col_md-6 col_sm-6 col_xs-12

slav-mirol replied:
Исправлено

slav-mirol committed:
modified:   main.js - добавлена функция перехода по нажатию кнопки на конкретный заголовок
	modified:   src/main.less - добавлены отступы в мобильную верстку
	deleted:    src/header copy.less
	deleted:    src/main copy.less
	deleted:    src/main.css
	modified:   src/utils/button.less - добавлены стили для различных состояний кнопки

m-somov commented on main.js:
```diff
@@ -1,2 +1,12 @@
 import 'normalize.css';
 import './index.less';
+
+const $button = document.querySelector('.button_scroll');
+const $topic = document.querySelector('.title_for_btn');
```
Можешь объяснить, зачем `$` в имени переменной?

slav-mirol replied:
По документации в JS можно использовать в названиях переменных $, я изменил название на cur_button и cut_topic соответственно

m-somov commented on index.html:
```diff
@@ -9,16 +9,318 @@
+          <section>
+            <div class="row">
+              <div class="col col_md-12 col_sm-12 col_xs-12">
+                <h2 class="title_for_btn">Как строится обучение</h2>
```
Лучше использовать id, тогда в JS можно будет получить элемент функцией getElementById, и не придется создавать новый css класс. Для кнопки тоже лучше создать id.

slav-mirol replied:
Исправлено

m-somov commented on src/main.less:
```diff
@@ -0,0 +1,168 @@
+  margin-bottom: 48px;
+}
+
+.title_for_btn {
+  .text-h2();
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  margin-top: 76px;
+  margin-bottom: 48px;
+}
```
Не нужно дублировать код, для таких случаев используют id. Плюс название не по БЭМ.

slav-mirol replied:
Добавил id для кнопки и топика, убрал повторяющийся код

m-somov commented on index.html:
```diff
@@ -9,16 +9,318 @@
+              Трехмесячный бесплатный учебный курс от Финтеха, который можно
+              проходить из любой точки России
+            </p>
+            <button class="button_scroll" type="button">Подробнее</button>
```
Ты изменил класс  `class="button"` -> `class="button_scroll"`. В итоге стили слетели. Нужно вернуть стили, и добавить новые для состояний hover, active, focus, как указано в макете в figma. Это можно сделать с помощью [псевдоклассов](https://developer.mozilla.org/ru/docs/Web/CSS/Pseudo-classes)

slav-mirol replied:
Исправлено

slav-mirol committed:
deleted:    src/header copy.less
	deleted:    src/main copy.less
	deleted:    src/main.css
	modified:   src/utils/button.less

slav-mirol committed:
modified:   index.html
	modified:   main.js
	modified:   src/main.less
	modified:   src/utils/button.less

slav-mirol committed:
modified:   index.html
	modified:   src/utils/button.less
